porcupine/plugins/highlight/tree_sitter_highlighter.py

```diff
@@ -140,6 +140,9 @@ def _decide_tag(self, node: tree_sitter.Node) -> str:
         else:
             type_name = node.type
 
+        if set(type_name) <= set("+-*/%~&|^!?<>="):
```

Comment:
- Why not include `@` here?
- Why not include `:` and the rest of punctuation here? Does it look bad in some themes if we map `Operator` and `Punctuation` to the same token tag?

Reply:
> Why not include @ here?

Idk, how other languages use `@`, so I thought it shouldn't be part of the "reasonable defaults"

I use the material theme, and there's no difference between `Operator` and `Punctuation`, I need to check it with other themes though

Reply:
I would prefer to just use the same color, at least for now:
- We don't need to distinguish between `Operator` and `Punctuation` for every supported language. This is important, because we both use color themes that highlight them the same.
- It's unclear whether `.` in Python, Java, C etc. should be `Operator` or `Punctuation`. It is a punctuation character but also an operator for accessing attributes.

Reply:
Actually it looks pretty weird with monokai, but it's my least favorite theme, so I don't really care
![image](https://user-images.githubusercontent.com/77941087/184530866-e6f762e1-65a6-4ee7-80c4-59599f546b39.png)